src/components/table/template.njk

```diff
@@ -0,0 +1,27 @@
+<table class="govuk-c-table {{ classes }}">
+  {% if caption %}
+  <caption class="govuk-c-table__caption {{captionSize}}">{{ caption }}</caption>
+  {% endif %}
+  {% if data.head %}
+  <thead class="govuk-c-table__head">
+    <tr class="govuk-c-table__row">
+    {% for label, type in data.head %}
+      <th class='govuk-c-table__header {% if type %} govuk-c-table__header--numeric {% endif %}' scope="col">{{ label }}</th>
+    {% endfor %}
+  </tr>
+  </thead>
+  {% endif %}
+  <tbody class="govuk-c-table__body">
+    {% for label, rows in data.rows %}
+      <tr class="govuk-c-table__row">
+        {% for label, cell in rows %}
+        {% if loop.first %}
+          <th class="govuk-c-table__header" scope="row">{{ label }}</th>
```

Comment:
- This assumes the first column will always be the heading – is that always going to be the case?
- The way this is structured means that the first row can never be of type `numeric`.
- Do we need to allow for other attributes to be specified per cell e.g. colspan, so that more complicated tables can be produced using this macro?